Astro 10/4 release notes (#1310)
astronomer/docs: astro/platform-variables.md

| `AIRFLOW__WEBSERVER__INSTANCE_NAME` | Shows the name of your Deployment in the Home view of the Airflow UI | `<Deployment-Name>` |
| `AIRFLOW__CELERY__WORKER_CONCURRENCY` | Determines how many tasks each Celery worker can run at any given time and is the basis of worker auto-scaling logic | `<Max-Tasks-Per-Worker>` |
| `AIRFLOW__WEBSERVER__NAVBAR_COLOR` | The color of the main navigation bar in the Airflow UI | `#4a4466` |

> | `AIRFLOW__WEBSERVER__EXPOSE_CONFIG=NON-SENSITIVE-ONLY` | Determines whether sensitive infomration about webserver configurations is exposed in the Airflow UI | `NON-SENSITIVE-ONLY` |

Reviewer: ```suggestion
| `AIRFLOW__WEBSERVER__EXPOSE_CONFIG` | Exposes the **Configuration** tab of the Airflow UI and hides sensitive values | `NON-SENSITIVE-ONLY` |
```
